src/index.html

```diff
@@ -1,13 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="
+            width=device-width,
+            user-scalable=no,
+            initial-scale=1.0,
+            maximum-scale=1.0,
+            minimum-scale=1.0
+    ">
```

Comment:
it is not really a mistake, but i just want to leave this here: the closing symbol should have the same indentation as the opening one. thats the general rule

```suggestion
    <meta name="viewport"
          content="
            width=device-width,
            user-scalable=no,
            initial-scale=1.0,
            maximum-scale=1.0,
            minimum-scale=1.0,
        "
    >
```

Reply:
but again, here you only have two attributes. you should start putting attributes each on its own line after their number is >= 3

Reply:
https://prnt.sc/1wpif5x

So, what I shoul do? Leave this line or formating? I don't really understand